### DISCUSSION TITLE
Bing的识图功能差不多公测完了……第三方客户端在跟进吗？

### DISCUSSION THREAD

**Discussion by Abcdefghuuj** [viopsa233/AI-testers-Forum#81] [Q&A]
一些疑问：
现在的第三方客户端不方便Bing看图，是不是又该改造一下chatsydney了？
_<img src="https://www.bing.com/images/blob?bcid=SJD1ea5Oi8oFZdpw4SA.Xifp1FmX.....5c">_
（侧边栏F12时看见的新块，似乎给bing看图用的，加完图后就有一个image地址保存图像）
但直接用文字发这一段Bing识别不出来……
微软是不是专门分出一个识别图片的bot来给bing上Prompt了？
以上，先等一波分析。

**Comment by ghost** [comment]
https://github.com/juzeon/SydneyQt/pull/37

**Reply by Abcdefghuuj** [reply]
原来有啊，没看见😅
不过有已经部署在网站上的吗，想白嫖[doge]

**Comment by SoraRoseous** [comment]
ChatSydney也可以了，不过我整得很烂将就用吧😂

**Reply by SoraRoseous** [reply]
可恶早知道有人已经做了我就去抄代码了，自己菜得一匹研究半天😅

**Reply by jonasreisner** [reply]
图片规制好像还挺严重的 稍微敏感点的图片就触发filter

**Reply by Abcdefghuuj** [reply]
呃，话说这filter是强制的还是可能被prompt抵消的那种？
